test/unit/mixins/evented.test.js

```diff
@@ -0,0 +1,455 @@
+/* eslint-env qunit */
+import sinon from 'sinon';
+import evented from '../../../src/js/mixins/evented';
+import * as Dom from '../../../src/js/utils/dom';
+import * as Obj from '../../../src/js/utils/obj';
+
+QUnit.module('mixins: evented', {
+
+  beforeEach() {
+    this.targets = {};
+  },
+
+  afterEach() {
+    Object.keys(this.targets).forEach(k => this.targets[k].trigger('dispose'));
+  }
+});
+
+QUnit.test('evented() mutations', function(assert) {
```

Comment:
I think that the name of this test should be something like "evented() An object is mutated as expected"